### ISSUE TITLE
Missing dependency for media-sound/lollypop

### ISSUE THREAD

**Issue by Bratzmeister** [jorgicio/jorgicio-gentoo#26]
Hello jorgicio,

I really like your effort for making an ebuild for lollypop.
When I installed it via portage there was a missing dependency though.

`virtual/python-sqllite` is not in the gentoo portage tree but the overlay "progress". In order to install your ebuild I had to add progress in layman which brings unstable python versions and modules with it.
I prefer not to have this. It would be very nice of you if you'd have an ebuild for virtual/python-sqllite in your overlay.

Thanks and keep up the good work!
Bratzmeister

**Comment by jorgicio** [comment]
All right! Fixed. Try again.

**Comment by Bratzmeister** [comment]
Fixed!
